release-1.8/release_notes_draft.md

```diff
@@ -221,3 +221,24 @@ kind.
 * [alpha] Support ipvs mode for kube-proxy([#46580](https://github.com/kubernetes/kubernetes/pull/46580), [@haibinxie](https://github.com/haibinxie))
 
 ## External Dependencies
+Continuous integration builds have used the following versions of external
+dependencies, however, this is not a strong recommendation and users should
+consult an appropriate installation or upgrade guide before deciding what
+versions of etcd, docker or rkt to use.
+
+- Docker versions 1.11.2, 1.12.6, 1.13.1 (17.03.2) have been validated
+    - Docker version 1.13.1 (17.03.2) [#42926](https://github.com/kubernetes/kubernetes/issues/42926)
+        - Shared PID namespace, live-restore and overlay2 are supported.
+        - The default iptables FORWARD policy has been changed from ACCEPT to
+          DROP, which causes outbound container traffic to stop working. See
+          [#40182](https://github.com/kubernetes/kubernetes/issues/40182) for
+          the workaround.
+        - The support for the v1 registries has been removed.
+    - Docker version 1.12.6 known issues
+        - Overlay2 is not supported
+        - Live-restore is not supported
+        - Shared pid namespace is not support
+    - Docker version 1.11.2 known issues
+        - kernel crash with Aufs storage driver on Debian Jessie (#27885) which can be identified by the node problem detector
+        - Leaked File descriptors (#275)
+        - Additional memory overhead per container (#21737)
```

Comment:
* Docker version 1.13.1 (17.03.2) #42926
    * Features
           * Shared PID namespace, live-restore, and overlay2 are supported.
           * The support for the v1 registries has been removed.
    * Known issues
            * The default iptables FORWARD policy has been changed from ACCEPT to DROP, which causes outbound container traffic to stop functioning. See #40182 for the workaround.

* Docker version 1.12.6 
    * Features
           * None
     * Known issues
            * Overlay2 is not supported
            * Live-restore is not supported
            * Shared pid namespace is not support

* Docker version 1.11.2 
     * Features
            * None
     * Known issues
            * kernel crashes with Aufs storage driver on Debian Jessie (#27885). This issue can be identified by the node problem detector
            * Leaked File descriptors (#275)
            * Additional memory overhead per container (#21737)

Reply:
Could you also provide a descriptive info on leaked file descriptors ?

Reply:
Updated the link.